src/main/java/de/zalando/aruha/nakadi/controller/EventTypeController.java

```diff
@@ -154,11 +157,24 @@ public EventTypeController(final EventTypeRepository eventTypeRepository, final
         }
     }
 
-    private void validatePartitioningStrategy(final EventType eventType) throws NoSuchPartitioningStrategyException {
+    private void validatePartitioningStrategy(final EventType eventType) throws NoSuchPartitioningStrategyException,
+            InvalidEventTypeException {
         final PartitionResolutionStrategy partitioningStrategy = eventType.getPartitionResolutionStrategy();
-        if (partitioningStrategy != null && !partitionResolver.strategyExists(partitioningStrategy.getName())) {
-            throw new NoSuchPartitioningStrategyException("partitioning strategy does not exist: " +
-                    partitioningStrategy.getName());
+        if (partitioningStrategy != null ) {
```

Comment:
Yes, null is allowed here. Because we agreed that partitioning strategy should be not required and 
1) We will have 'random' strategy as a default together with:
2) We will have 1 partition for EventType by default (which is already there)